yt/yt/server/node/cluster_node/proxying_chunk_service.h

```diff
@@ -15,7 +15,8 @@ namespace NYT::NClusterNode {
 NRpc::IServicePtr CreateProxyingChunkService(
     NElection::TCellId cellId,
     TProxyingChunkServiceConfigPtr serviceConfig,
-    NApi::NNative::TMasterConnectionConfigPtr connectionConfig,
+    NApi::NNative::TMasterConnectionConfigPtr masterConnectionConfig,
+    NApi::NNative::TConnectionDynamicConfigPtr connectionConfig,
```

Comment:
How about `nativeConnectionConfig` for better distinction?

Reply:
Good idea!